pkg/fixtures/dackbox.go

```diff
@@ -0,0 +1,1162 @@
+			Availability:        storage.CVSSV3_IMPACT_HIGH,
+			Score:               7.8,
+			Severity:            storage.CVSSV3_HIGH,
+		},
+		PublishedOn:       &types.Timestamp{Seconds: 1234567890},
+		LastModified:      &types.Timestamp{Seconds: 1235467890},
+		VulnerabilityType: storage.EmbeddedVulnerability_IMAGE_VULNERABILITY,
+		VulnerabilityTypes: []storage.EmbeddedVulnerability_VulnerabilityType{
+			storage.EmbeddedVulnerability_IMAGE_VULNERABILITY,
+		},
+		Suppressed:            false,
+		SuppressActivation:    nil,
+		SuppressExpiry:        nil,
+		FirstSystemOccurrence: &types.Timestamp{Seconds: 1243567890},
+		FirstImageOccurrence:  &types.Timestamp{Seconds: 1245367890},
+		Severity:              storage.VulnerabilitySeverity_LOW_VULNERABILITY_SEVERITY,
+		State:                 storage.VulnerabilityState_OBSERVED,
+	}
+}
+
+// GetEmbeddedImageCVE_2345_0007 provides a pseudo-realistic image CVE for dackbox datastore integration testing.
+func GetEmbeddedImageCVE_2345_0007() *storage.EmbeddedVulnerability {
+	return &storage.EmbeddedVulnerability{
+		Cve:          "CVE-2345-0007",
+		Cvss:         5.9,
+		Summary:      "Find some inspiring quote on an evil topic to insert here.",
+		Link:         "book://author/title",
+		SetFixedBy:   &storage.EmbeddedVulnerability_FixedBy{FixedBy: ""},
+		ScoreVersion: storage.EmbeddedVulnerability_V3,
+		CvssV2: &storage.CVSSV2{
+			Vector:              "AV:N/AC:M/Au:N/C:N/I:P/A:N",
+			AttackVector:        storage.CVSSV2_ATTACK_NETWORK,
+			AccessComplexity:    storage.CVSSV2_ACCESS_MEDIUM,
+			Authentication:      storage.CVSSV2_AUTH_NONE,
+			Confidentiality:     storage.CVSSV2_IMPACT_NONE,
+			Integrity:           storage.CVSSV2_IMPACT_PARTIAL,
+			Availability:        storage.CVSSV2_IMPACT_NONE,
+			ExploitabilityScore: 8.6,
+			ImpactScore:         2.9,
+			Score:               4.3,
+			Severity:            storage.CVSSV2_MEDIUM,
+		},
+		CvssV3: &storage.CVSSV3{
+			Vector:              "CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:H/A:N",
+			ExploitabilityScore: 2.2,
+			ImpactScore:         3.6,
+			AttackVector:        storage.CVSSV3_ATTACK_NETWORK,
+			AttackComplexity:    storage.CVSSV3_COMPLEXITY_HIGH,
+			PrivilegesRequired:  storage.CVSSV3_PRIVILEGE_NONE,
+			UserInteraction:     storage.CVSSV3_UI_NONE,
+			Scope:               storage.CVSSV3_UNCHANGED,
+			Confidentiality:     storage.CVSSV3_IMPACT_NONE,
+			Integrity:           storage.CVSSV3_IMPACT_HIGH,
+			Availability:        storage.CVSSV3_IMPACT_NONE,
+			Score:               5.9,
+			Severity:            storage.CVSSV3_MEDIUM,
+		},
+		PublishedOn:       &types.Timestamp{Seconds: 1234567890},
+		LastModified:      &types.Timestamp{Seconds: 1235467890},
+		VulnerabilityType: storage.EmbeddedVulnerability_IMAGE_VULNERABILITY,
+		VulnerabilityTypes: []storage.EmbeddedVulnerability_VulnerabilityType{
+			storage.EmbeddedVulnerability_IMAGE_VULNERABILITY,
+		},
+		Suppressed:            false,
```

Comment:
<strike>Would it be worthwhile to add suppressed CVE's as well?</strike>

I see we haven't implemented that yet.

Reply:
Good point.
I think I'd use the datastore Suppress/Unsuppress functions to test the behaviour, though.